docs/DeveloperGuide_Winery.md

```diff
@@ -0,0 +1,622 @@
+---
+{}
+---
+---
+---
+<center>
+
+![alt text](graphics/Figure00-Cloudcycle.png "Cloudcyle")
+
+###Bereitstellung, Verwaltung und vermarktung von portablen Cloud-Diensten mit  
+###garantierter Sicherheit und Compliance während des gesamten lebenszyklus
+
+</center>
+<center>
+
+#Winery - Developer Guide
+##Version 1.0
+
+</center>
+
+---
+
+|                     |                                        | 
+|---------------------|----------------------------------------|
```

Comment:
Remove the whole table.

In Git, version control works different from Word version control.